test/test_main.py

```diff
@@ -7,7 +7,7 @@ class TestMain(unittest.TestCase):
     @patch('bowser.main.main')
     def test__init_calls_main_once(self, mock_main):
         with patch.object(bowser.main, '__name__', '__main__'):
-            bowser.main.init()
+            bowser.main.main()
             mock_main.assert_called_once_with()
```

Comment:
This test will always pass now.  Previously I tried to run the main.py file with `__name__` set to `__main__` to trigger the main method to be called even though it was patched.  But, it would always call the real main function